js-miniapp-sdk/README.md

```diff
@@ -411,6 +411,54 @@ miniApp.chatService.sendMessageToMultipleContacts(messageToContact)
 
 ## Advanced Usage
 
+### Errors management
+
+Error messages sent to the bridge should always be of the format: `"error_type_key: error_message"` or `"error_key"`.
+The SDK will throw a `MiniAppError` extracting the `error_type_key` from this message and check if an error message exists locally. 
+If no error message exists, it will then construct one base on `error_message` if available.
+If the error type key is not supported, `MiniAppError.type` will automatically be set to `Other`
+Here are the currently supported type keys:
+
+| Error Type | Message |
+| ---- | ---- |
+| `AudienceNotSupportedError` | `The value passed for 'audience' is not supported.` |
+| `ScopesNotSupportedError` | `The value passed for 'scopes' is not supported.` |
+| `AuthorizationFailureError` | _a message should be provided_ |
+| `Other` |  _a message should be provided_ |
+
+
+Here is an example of how `MiniAppError` is populated if the bridge receives a valid key `AudienceNotSupportedError`:
+
+```javascript
+miniApp.user.getAccessToken("TOKEN_AUDIENCE", ["TOKEN_SCOPE1","TOKEN_SCOPE2"])
+  .then(data => {
+      ...
+  })
+  .catch(error => {
+      console.error(error.name);     // AudienceNotSupportedError
+      console.error(error.message); //  The value passed for 'audience' is not supported.
+      console.error(error.type);   //   AudienceNotSupportedError
```

Comment:
I think we should have an example of how they would handle different types of errors. For example checking the `error.type` - or if you change it to use error classes as I mentioned in the other comment:
```
if (error instanceof AuthorizationFailureError) {
    // handle error
}
```

Reply:
got it